### ISSUE TITLE
Failed to create a cf manifest using make_manifest_spiff script

### ISSUE THREAD

**Issue by MarikoTagawa** [cloudfoundry/bosh-lite#249]
Hi, I'm trying to deploy Cloud Foundry on my Redhat 6.5 PC.

To create a cf manifest, I ran the following command and got an error and it's failed. 
Anybody has any idea how to fix the issue? 
------------------------------------------------------------------------------------------------
$ ./bin/make_manifest_spiff
a8e0f903-5202-48f9-a683-6ab5114a3aeb
2015/04/05 01:25:07 error reading stub [/home/mtagawa/workspace/cf-release/templates/cf-lamb.yml]: open /home/mtagawa/workspace/cf-release/templates/cf-lamb.yml: no such file or directory
Incorrect YAML structure in `/home/mtagawa/workspace/bosh-lite/manifests/cf-manifest.yml': expected Hash at the root
Config
/home/mtagawa/.bosh_config
------------------------------------------------------------------------------------------------
I checked the file;cf-lamb.yml. It's linked to the following file and the file doesn't exist. 
lrwxrwxrwx.  1 mtagawa mtagawa    49 Apr  4 23:00 cf-lamb.yml -> ../src/loggregator/manifest-templates/cf-lamb.yml

My environment is here
----------------------------------------------------------------------
- H/W
	- Lenovo Thinkpad X220
	- Processor: Intel(R) Core(TM) i5-2520M CPU @ 2.50GHz
	- CPU core:2
	- Processor#:4
	- Memory: 16GB
	- Host OS: Red Hat Enterprise Linux Workstation release 6.5

- S/W
 - Prerequired s/w install
	- VirtualBox Manager 4.3.26
	- git version 1.7.1
	- ruby 2.2.1p85
	- Vagrant 1.7.2
	- spiff version 1.0.3
---------------------------------------------------------------------------------------------------

Implementation steps:

- Install BOSH Lite Steps:
1. Intall bash_cli
	- BOSH 1.2905.0
2. Clone this repository
3. Install and boot a Virtual Machine. Provider = Virtualbox
4. Start Vagrant from the base directory
5. Target the BOSH Director and login the VM as admin/admin
6. Add a set of route entries

- Deploy Cloud Foundry
1. Clone the cf-release repository into the same directory that I cloned this repository
2. Manual deploy
3. Upload release cf-204.yml
4. Upload the Warden stemcell
5. Use the make_manifest_spiff script to create a cf manifest.

**Comment by pivotal-graeme-davison** [comment]
Hi Mariko,

Did you try a git submodule update --init --recursive in the cf-release directory?  I ran in to a similar problem and I believe this is what I was forgetting to do.

- Graeme

**Comment by MarikoTagawa** [comment]
Hi. pivotal-graeme-davison.
Thanks for your comment. Yes, I ran "update" again and again but the directory: ~workspace/cf-release/src/loggregator was empty. 

So I went to https://github.com/cloudfoundry/loggregator and downloaded loggregator-develop.zip and extracted under ~workspace/cf-release/src/loggregator.
Then $ ./bin/make_manifest_spiff worked and generated manifests/cf-manifest.yml. 

I'm not sure why 'update' command didn't retrieve files under ~workspace/cf-release/src/loggregator.
It's because my OS is Redhat or anything else. Anyway I went through the issue and successfully deployed CF.
